extensions/2.0/Vendor/EXT_mesh_primitive_edge_visibility/README.md

```diff
@@ -0,0 +1,221 @@
+```
+{
+    "lineStrings": [
+        {
+            "indices": 5
+        },
+        {
+            "indices": 6,
+            "material": 1
+        }
+    ]
+}
+```
+
+### Silhouette Mates
+
+A silhouette edge represents the edge between two vertices shared by a pair of adjacent triangles. An engine can directly obtain the indices of one of those triangles based on the `visibility` encoding, but to conditionally render the silhouette it also needs to know the third (unshared) vertex of the other triangle so that it can determine each triangle's facing relative to the camera.
+
+The extension's `silhouetteMates` property specifices the index of an accessor of `SCALAR` type and component type `UNSIGNED_BYTE` (5121), `UNSIGNED_SHORT` (5123), or `UNSIGNED_INT` (5125) encoding indices into the triangle mesh's vertex attribute array. Each index refers to the unshared vertex of the other triangle adjacent to the `nth` silhouette edge encoded in `visibility`. The accessor's `count` **MUST** be equal to the number of edges encoded with visibility value `1`.
+
+The `silhouetteMates` property **MUST** be defined *if and only if* at least one edge is encoded with visibility value `1` in `visibility`.
+
+Engines **MUST** render a silhouette edge unless both adjacent triangles are front-facing or both are back-facing, as determined by their normal vectors.
+
+## Rendering requirements
+
+- Engines **MUST** render all edges according to their specified visibility values, though some engines may permit the user to toggle the display of edges on and off.
+- The edges **MUST** draw in front of their corresponding triangles with no depth-fighting.
+- The edges **MUST** draw using the materials specified by the extension by default, though some engines may provide options for the user to override those materials.
+- Each silhouette edge **MUST** be rendered unless both adjacent triangles are front-facing or both are back-facing, as determined by their normal vectors.
+
+## JSON Schema
+
+- [primitive.EXT_mesh_primitive_edge_visibility.schema.json](./schema/primitive.EXT_mesh_primitive_edge_visibility.schema.json)
+- [lineString.schema.json](./schema/lineString.schema.json)
+
+## Implementation Notes
+
+The [pull request](https://github.com/iTwin/itwinjs-core/pull/5581) that informed the design of this extension provides [an iterator](https://github.com/iTwin/itwinjs-core/blob/03b760e1e91bde5221aa7370ea45c52f966e3368/core/frontend/src/common/imdl/CompactEdges.ts#L42) over the `visibility` buffer.
+
+iTwin.js [implements](https://github.com/iTwin/itwinjs-core/blob/03b760e1e91bde5221aa7370ea45c52f966e3368/core/frontend/src/internal/render/webgl/glsl/Edge.ts#L107) conditional display of silhouette edges. It also draws edges in a separate pass from surfaces to [ensure](https://github.com/iTwin/itwinjs-core/blob/03b760e1e91bde5221aa7370ea45c52f966e3368/core/frontend/src/internal/render/webgl/glsl/FeatureSymbology.ts#L426) the mitigate z-fighting.
```

Comment:
Thanks for the links. I think we can do this in CesiumJS with a separate pass after the `OPAQUE` pass.